org.gustavoperedo.FontDownloader.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id" : "org.gustavoperedo.FontDownloader",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "fontdownloader",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=~/.local/share/fonts",
+        "--filesystem=~/Downloads"
```

Comment:
```suggestion
        "--filesystem=xdg-download"
```

Reply:
For now the application only downloads tp the ~/Downloads location and creates it if it doesn't exist. I'm planning on the next version to have a file choose for the person to actually save where they want it to be saved, but for now the code only downloads to ~/.local/share/fonts and ~/Downloads

Reply:
Why hardcoding it to Downloads instead of using the xdg dirs?

Reply:
Because I didn't know how to use xdg standarts in code... Well, I guess that will indeed help user experience.

Can you give a look at the rest of the json file to see if there is anything else missing? This way I won't botter you as much.